core/src/main/java/com/airbnb/aerosolve/core/function/Function.java

```diff
@@ -5,7 +5,7 @@
 import java.io.Serializable;
 import java.util.List;
 
-public interface Function extends Serializable {
+public interface Function extends Serializable, Cloneable {
```

Comment:
usually copy constructor is favor over clone, how about  Function deepCopy() in the interface